ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionBuilderModule.java

```diff
@@ -164,9 +168,21 @@ public SafeFuture<HeaderWithFallbackData> builderGetHeader(
                         .map(ExecutionPayloadWithValue::getValue)
                         .orElse(UInt256.ZERO);
                 logReceivedBuilderBid(signedBuilderBid.getMessage());
-                if (signedBuilderBid.getMessage().getValue().lessOrEqualThan(localPayloadValue)) {
+                // 1 ETH is 10^18 wei, Uint256 max is more than 10^77
+                if (builderBidChallengePercentage.isPresent()
+                    && signedBuilderBid
+                        .getMessage()
+                        .getValue()
+                        .multiply(builderBidChallengePercentage.get())
+                        .lessOrEqualThan(localPayloadValue.multiply(HUNDRED_PERCENTS))) {
+                  LOG.info(
+                      "Using local Execution Payload instead of Builder Bid as it's challenged. "
+                          + "Builder bid value: {}, local block value: {}, builderBidChallengePercentage: {}%",
+                      signedBuilderBid.getMessage().getValue().toDecimalString(),
+                      localPayloadValue.toDecimalString(),
+                      builderBidChallengePercentage);
                   return getResultFromLocalExecutionPayload(
-                      localExecutionPayload, slot, FallbackReason.LOCAL_BLOCK_VALUE_HIGHER);
+                      localExecutionPayload, slot, FallbackReason.LOCAL_BLOCK_VALUE_WIN);
```

Comment:
what about `LOCAL_BLOCK_VALUE_WON` ? similarly to `CIRCUIT_BREAKER_ENGAGED`

Reply:
it was the first name. don't know why I reverted it